src/components/FeatureViews/AdminPortal/AdminUtils.js

```diff
@@ -0,0 +1,173 @@
+import { isSunday } from "date-fns";
+
+export const roleColor = (string)=>{
+    switch(string){
+        case "administrator":
+            return "red";
+        case "parent":
+            return "#812CD7";
+        case "student":
+            return "#378BFF";
+        case "instructor":
+            return "#1FA18A"
+        case "receptionist":
+            return "#D04E3D"
+        default:
+            return "no match"
+    }
+}
```

Comment:
Since the default should never run (a match should be found), this can be concisely written as an object:

     export const roleColor = {
        "administrator": "red",
        "parent": "#812CD7",
        // similar for the rest
    };

(note that you will then have to use it as roleColor[role] instead of with parentheses)